app/views/users/index.html.erb

```diff
@@ -8,6 +8,7 @@
   <div class="init-list">
   <% @users.each do |user| %>
     <div class="card">
+      <%= link_to edit_user_path(user), :style=> "text-decoration: none;" do %>
```

Comment:
Do we need this edit link?

Reply:
This link was removed. It was not needed